docs/gds-supported-platform/gsp-service-operator.md

```diff
@@ -28,9 +28,35 @@ items:
 ```
 This will create an SQS Queue on AWS named alexs-test-queue, with a message retention period of 1 hour, and a maximum message size of 1KiB. It will also ensure you can get access to the created queue. It will store the queue URL in a secret named `alexs-test-queue-secret` that we will use below.
 
+Here's an example of a Postgres database:
```

Comment:
Looks like some YAML to me. :trollface:

Reply:
You'll need to have blank lines at the start and end of code blocks to successfully interpret them as code.

Reply:
```suggestion
Here's an example of a Postgres database:

```